docs/environments/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Environments
+intro: Use environments to organize the infrastructure on which your application will run, provision new cloud resources, configure network, install operating system services and have fine grained control of your application deployments.
```

Comment:
```suggestion
intro: Use environments to organize the infrastructure on which your application will run, provision new cloud resources, manage operating system resources and have fine grained control of your application deployments.
```
Tried to use `manage operating system resources` instead of `configure network, install operating system services` because i believe its covers more resources (cronjob, ssh-keys, network rules, service and daemons).

Reply:
Thanks! As network configuration requires management on both OS level and infra level, what about the wording below?
```suggestion
intro: Use environments to organize the infrastructure on which your application will run, provision new cloud resources, manage operating system resources, network configuration and have fine grained control of your application deployments.
```